tracer/src/Datadog.Trace/IDatadogTracer.cs

```diff
@@ -18,6 +18,8 @@ internal interface IDatadogTracer
 
         IGitMetadataTagsProvider GitMetadataTagsProvider { get; }
 
+        IExtraServicesProvider ExtraServicesProvider { get; }
```

Comment:
It was required to be able to access the service from the `TraceContext` but since I changed the way service accessed because of other comments it's no longer necessary, see 14f828b6ca89a2f776967e35a647b6493ec40c97